.changeset/slimy-weeks-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/edr": minor
```

Comment:
Does this affect the npm package to merit a version bump?

Reply:
I mean, you can't release it under the same version, but I guess it could be a patch version.

Reply:
Yeah. I don't have any preference tbh, was mainly curious.